src/components/VirtualThought.tsx

```diff
@@ -144,7 +144,12 @@ const VirtualThought = ({
     const autofocusNew = calculateAutofocus(store.getState(), path)
     const isVisibleNew = autofocusNew === 'show' || autofocusNew === 'dim'
     if (!ref.current) return
-    const heightNew = ref.current.getBoundingClientRect().height
+
+    // Need to grab max height between .thought and .thought-annotation since the annotation height might be bigger (due to wrapping link icon).
+    const heightNew = Math.max(
+      ref.current.getBoundingClientRect().height,
+      ref.current.querySelector('.thought-annotation')?.getBoundingClientRect().height || 0,
```

Comment:
Does this work if you just measure the height of the ThoughtAnnotation instead of the Thought? (instead of taking the max of the two)

Reply:
Yes, the Thought height will always be the same or less than ThoughtAnnotation.

Reply:
I think we should really optimize this query since it is a performance bottleneck. A direct ref to the ThoughtAnnotation would be ideal. Otherwise a fixed DOM traversal (admittedly fragile but probably worth it in this case).

Reply:
We have some cases where we don't have .though-annotation. 
![image](https://github.com/user-attachments/assets/67af26d6-b746-4263-86a3-c9be9ef64d50)

Reply:
How about we do this? 
```    let heightNew = ref.current.querySelector('.thought-annotation')?.getBoundingClientRect().height || 0
    if (heightNew == 0) {
      heightNew = ref.current.getBoundingClientRect().height
    }
```

Reply:
> We have some cases where we don't have .though-annotation.

Good catch.

> How about we do this?

Sure. Let's do that and optimize the `.thought-annotation` selector.

Except I'm now seeing that this does not result in the correct height. (Careful with a quick fix of a px offset here; the height needs to work across all font sizes.)

## Current Behavior

a4f9b63c78

<img width="742" src="https://github.com/user-attachments/assets/203d7bf5-cad5-41e5-ad1b-5f185b2beb38">

## Expected Behavior

`main`

<img width="746" src="https://github.com/user-attachments/assets/dfddd166-54ec-4ae1-b1fb-51f0dff25d1a">

Reply:
@baproactive